packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -48,8 +50,18 @@ export async function bundleServer(buildOpts: BuildOptions): Promise<void> {
     format: "esm",
     target: "esnext",
     minify: false,
-    plugins: [createFixRequiresESBuildPlugin(buildOpts)],
-    external: ["./middleware/handler.mjs", "caniuse-lite"],
+    plugins: [
+      createFixRequiresESBuildPlugin(buildOpts),
+      inlineRequirePagePlugin(buildOpts),
+      setWranglerExternal(),
+    ],
+    external: [
+      "./middleware/handler.mjs",
+      // Next optional dependencies.
```

Comment:
note: 
I plan to refactor how optional dependencies are handled.
That is put them all in a single array, add comments and use that here and pass them to `patchOptionalDependencies()`